### PR TITLE
[OGUI-1414] Retrieve initial data needed for quering and building calibration page

### PR DESCRIPTION
**Pull request by graduta**
#### I have JIRA issue created
- [x] branch and/or PR name(s) includes JIRA ID
- [x] issue has "Fix version" assigned
- [x] issue "Status" is set to "In review"
- [x] PR labels are selected
- [x] FLP integration tests were ran successful

PR which:
* extracts constants used in KV store retrieval of information so that they can be reused by services
* adds new `CalibrationRunService` which will be the high-level service used for building data needed for the Calibration Page
* moves the storing of `runTypes` from `BookkeepingService` so that it is kept as a low service
* adds new enum file for storing `logLevels`

### CODE REVIEW

File: Control/lib/services/Bookkeeping.service.js [hunk 1]
```diff
@@ -33,40 +33,29 @@ class BookkeepingService {
...
-   */
-  async init() {
-    this._runTypes = await this._getRunTypes();
-  }
-
   /**
    * Given a definition, a type of a run and a detector, fetch from Bookkeeping the last RUN matching the parameters
    * @param {String} definition - definition of the run to query
-   * @param {String} type - type of the run to query
+   * @param {Number} type - type of the run to query as per BKP mapping
```
Comment (martinboulais) [inline Control/lib/services/Bookkeeping.service.js]: I would put typeId if it's the id

File: Control/lib/services/CalibrationRun.service.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+      /**
+       * @type {Object<String, Array<String>>}
+       * @example 
+       * { "XYZ": ["CALIB1", "CALIB2"], "ABC": ["XCALIB"] }
+       */
+      this._calibrationPerDetectorMap = await this._apricotService.getRuntimeEntryByComponent(COG, CALIBRATION_MAPPING);
+    } catch (error) {
+      const err = grpcErrorToNativeError(error);
+      this._logger.errorMessage(`Unable to load calibration mapping due to: ${err}`,
+        {level: 5, system: 'GUI', facility: 'calibration-service'}
```
Comment (martinboulais) [inline Control/lib/services/CalibrationRun.service.js]: Don't we have constants for level? We should

**Review by martinboulais — CHANGES_REQUESTED** [review]

File: Control/lib/services/Bookkeeping.service.js (hunk 1 above)
Comment (graduta) [reply Control/lib/services/Bookkeeping.service.js]: Updated, I missed it when migrating from String to Number
Thank you!

File: Control/lib/services/CalibrationRun.service.js (hunk 2 above)
Comment (graduta) [reply Control/lib/services/CalibrationRun.service.js]: Yes, we should and the plan is to add them in the `webui/framework` with the new logger library.
For now, I have added them as constants

**Review by martinboulais — APPROVED** [review]

**Pull request merged**